router/index.js

```diff
@@ -5,43 +5,43 @@ const express = require('express');
 const router = express.Router();
 
 // Register route
```

Comment:
Good placement of comments. As the routes expand, clear comments will be important for readability.